plugins/inputs/logfile/tail/tail.go

```diff
@@ -510,8 +510,13 @@ func (tail *Tail) sendLine(line string, offset int64) bool {
 		select {
 		case tail.Lines <- &Line{line, now, nil, offset}:
 		case <-tail.Dying():
-			tail.dropCnt += len(lines) - i
-			return true
+			if tail.Err() == errStopAtEOF {
+				// Try sending, even if it blocks.
+				tail.Lines <- &Line{line, now, nil, offset}
```

Comment:
🤔  maybe this is just such an extreme case it wouldn't happen, but don't we want to return something here? Part of me thinks that it doesn't matter since this only gets executed when the tail routine is `Dying()`.

Reply:
This is inside a for loop, although it typically loops just once, unless the line is huge.
Regardless, no I don't think we need to return.
The return type of this method is `bool` and there is a `return true` at the bottom of the method.
If the `Tail` is dying because of `errStopAtEOF` we want to act as if it is NOT dying at all.